src/main/interface/cli.c

```diff
@@ -489,10 +489,13 @@ static void cliPrintVarRange(const clivalue_t *var)
     case (MODE_LOOKUP): {
         const lookupTableEntry_t *tableEntry = &lookupTables[var->config.lookup.tableIndex];
         cliPrint("Allowed values:");
-        for (uint32_t i = 0; i < tableEntry->valueCount ; i++) {
-            if (i > 0)
-                cliPrint(",");
-            cliPrintf(" %s", tableEntry->values[i]);
+        for (uint32_t i = 0; i < tableEntry->valueCount; i++) {
```

Comment:
(`unsigned` instead of `uint32_t` would be cleaner ... )

Reply:
We use `size_t` in such places I think

Reply:
I was just reusing existing code (at 2 am), but good points, will fix up in another pull request.